New Wix Doc (#1262)
mercadopago/devsite-docs: guides/plugins/unofficial/wix.en.md

# Wix

> Wix is ​​a platform that allows you to create, design, manage and develop an online store in which you can offer your store's customers the possibility of making payments using Mercado Pago [Checkout Pro](https://www.mercadopago[FAKER][URL][DOMAIN]/ferramentas-para-vender/cobrar?utm_experiment=optimize&matt_tool=69348836&matt_word=MLB_MP_G_XP_OP_COW_SEARCH_SELL_TXS_Checkout-Transparente-Branded-NovaLP&gclid=CjwKCAjw3_KIBhA2EiwAaAAlinz2stvLcjAuKbXruVqdJtI_cAemAj6CaP3Q3yxbxbiumdlbgMsynhoCeLsQAvD_BwE).

Reviewer: ```suggestion
Wix is ​​a platform that allows you to create, design, manage and develop an online store in which you can offer your customers the possibility of making payments using Mercado Pago [Checkout Pro](https://www.mercadopago[FAKER][URL][DOMAIN]/ferramentas-para-vender/cobrar?utm_experiment=optimize&matt_tool=69348836&matt_word=MLB_MP_G_XP_OP_COW_SEARCH_SELL_TXS_Checkout-Transparente-Branded-NovaLP&gclid=CjwKCAjw3_KIBhA2EiwAaAAlinz2stvLcjAuKbXruVqdJtI_cAemAj6CaP3Q3yxbxbiumdlbgMsynhoCeLsQAvD_BwE).
```

Reviewer: Dejo sugerencia por repetición de "store".

With **Checkout Pro**, your customer is directed to the Mercado Pago website to make the payment and is redirected to your store as soon as the payment is done. All this simply, quickly and securely. 
 

## Requirements
In order to offer Mercado Pago as a payment method in your Wix online store, you will need to have the items described below.

>  - **Mercado Pago Account**: when you have a Mercado Pago account, you will have access to your credentials. You will need them to set up the Mercado Pago connection as a payment provider in your Wix store. If you don't have one yet, just [create](https://www.mercadopago[FAKER][URL][DOMAIN]/hub/registration/landing).

Reviewer: ```suggestion
 - **Mercado Pago account**: when you have a Mercado Pago account, you will have access to your credentials. You will need them to set up the Mercado Pago connection as a payment provider in your Wix store. If you don't have one yet, just [create](https://www.mercadopago[FAKER][URL][DOMAIN]/hub/registration/landing).
```

>NOTE
>
>Important
>
>Want to learn more about credentials? Find more information under [credentials](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/resources/credentials).

## Connection and configuration

> Connecting and configuring Mercado Pago as a payment method in your Wix store is done in two simple and quick steps. In the first, you connect Mercado Pago as a payment method in your Wix store and in the second, you configure the connector with your production credentials.

Reviewer: ```suggestion
Connecting and configuring Mercado Pago as a payment method in your Wix store can be done in two simple and quick steps. In the first, you connect Mercado Pago as a payment method in your Wix store and in the second, you configure the connector with your production credentials.
```

> 1. Make the connection as described in [this Wix documentation](https://support.wix.com/en/article/conectando-mercadopago-como-provedor-de-pagamento) You will automatically be taken to the connector's configuration page.

Reviewer: ```suggestion
1. Make the connection as described in [this Wix documentation](https://support.wix.com/en/article/conectando-mercadopago-como-provedor-de-pagamento) You will automatically be redirected to the connector's configuration page.
```
